src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -53,7 +53,8 @@ internal partial class Binder
             bool isCast,
             bool wasCompilerGenerated,
             TypeSymbol destination,
-            DiagnosticBag diagnostics)
+            DiagnosticBag diagnostics,
+            bool? isNullable = null)
         {
```

Comment:
What does this mean? I suspect it has nothing to do with the current "nullable conversions" in C# 6. If true, please rename and document this.